modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/service/test/NotificationTemplateServiceTest.java

```diff
@@ -169,6 +169,7 @@ private NotificationContext _createNotificationContext(User user)
 		notificationTemplate.setEditorType(
 			NotificationTemplateConstants.EDITOR_TYPE_RICH_TEXT);
 		notificationTemplate.setName(RandomTestUtil.randomString());
+		notificationTemplate.setSubject("Subject [%term%]");
```

Comment:
Could you change "Subject [%term%]" to RandomTestUtil.randomString(), please?
If it doesn't matter to the test, we should make things random :+1:

Reply:
Ok, working on it! :)